src/test/javascript/spec/component/tutorial-groups/crud/create-tutorial-group.component.spec.ts

```diff
@@ -0,0 +1,107 @@
+            imports: [],
+            declarations: [CreateTutorialGroupComponent, LoadingIndicatorContainerStubComponent, TutorialGroupFormStubComponent, MockPipe(ArtemisTranslatePipe)],
+            providers: [
+                MockProvider(TutorialGroupsService),
+                MockProvider(AlertService),
+                { provide: Router, useClass: MockRouter },
+                {
+                    provide: ActivatedRoute,
+                    useValue: {
+                        parent: {
+                            paramMap: of({
+                                get: (key: string) => {
+                                    switch (key) {
+                                        case 'courseId':
+                                            return 1;
+                                    }
+                                },
+                            }),
+                        },
+                    },
+                },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                createTutorialGroupComponentFixture = TestBed.createComponent(CreateTutorialGroupComponent);
+                createTutorialGroupComponent = createTutorialGroupComponentFixture.componentInstance;
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should initialize', () => {
+        createTutorialGroupComponentFixture.detectChanges();
+        expect(createTutorialGroupComponent).not.toBeNull();
+    });
+
+    it('should send POST request upon form submission and navigate', fakeAsync(() => {
+        const router: Router = TestBed.inject(Router);
+        const tutorialGroupService = TestBed.inject(TutorialGroupsService);
+        const exampleTeachingAssistant = new User();
+        exampleTeachingAssistant.login = 'testLogin';
+
+        const formDate: TutorialGroupFormData = {
+            title: 'Test',
+            teachingAssistant: exampleTeachingAssistant,
+        };
+
+        const response: HttpResponse<TutorialGroup> = new HttpResponse({
+            body: new TutorialGroup(),
+            status: 201,
+        });
+
+        const createStub = jest.spyOn(tutorialGroupService, 'create').mockReturnValue(of(response));
+        const navigateSpy = jest.spyOn(router, 'navigate');
+
+        createTutorialGroupComponentFixture.detectChanges();
+        tick();
+        const tutorialGroupForm: TutorialGroupFormStubComponent = createTutorialGroupComponentFixture.debugElement.query(
+            By.directive(TutorialGroupFormStubComponent),
+        ).componentInstance;
+        tutorialGroupForm.formSubmitted.emit(formDate);
+
+        createTutorialGroupComponentFixture.whenStable().then(() => {
+            const tutorialGroupCallArgument: TutorialGroup = createStub.mock.calls[0][0];
+
+            expect(tutorialGroupCallArgument.title).toEqual(formDate.title);
+
+            expect(createStub).toHaveBeenCalledOnce();
```

Comment:
Missing `toHaveBeenCalledWith()`

Reply:
Not needed as I check the call Argument above